coderadar-ui/src/app/view/project-dashboard/list-view/list-view.component.html

```diff
@@ -35,7 +35,7 @@
   </span>
 
   <div class="analysisButtons">
-    <button title="Start analysis for the chosen branch" (click)="startAnalysis(project.id, selectedBranch)" style="float:right" mat-icon-button>
+    <button title="Start analysis for the chosen branch" (click)="startAnalysis(project.id)" style="float:right" mat-icon-button>
```

Comment:
Please change the title of this button, since all branches are analyzed, no matter which branch is chosen in the UI.

Reply:
Done